tools/test-tools/package.json

```diff
@@ -34,14 +35,14 @@
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.51.0",
 		"@fluidframework/eslint-config-fluid": "^5.6.0",
-		"@types/mocha": "^10.0.10",
+		"@types/mocha": "^9.1.1",
 		"@types/node": "^18.19.0",
 		"eslint": "~8.55.0",
 		"eslint-config-prettier": "~9.0.0",
-		"mocha": "^10.8.2",
+		"mocha": "^10.2.0",
 		"mocha-multi-reporters": "^1.5.1",
 		"prettier": "~3.0.3",
-		"rimraf": "^5.0.0",
+		"rimraf": "^4.4.0",
```

Comment:
Update dependencies to match the versions used in the rest of the client release group. Note that it meant a few deps went to an earlier version, but should be ok.